python/src/server.py

```diff
@@ -0,0 +1,53 @@
+# from typing import Any, List, Literal, Optional, Union
```

Comment:
Can we change this file name from server to something else? and add another file for initializing the server, so that user can focus only on their model, and not about anything server-related.

Reply:
The other file will then simply initialize the `app` that `uvicorn` will expect (will be one line of code), so I would rather leave all the implementation in one file.